doc/cephfs/index.rst

```diff
@@ -42,9 +42,21 @@ Block Devices) and RGW (Ceph Object Storage Gateway).
 Using CephFS
 ============
 
-Using the Ceph File System requires at least one :term:`Ceph Metadata Server` in
-your Ceph Storage Cluster.
+Using CephFS with a running Ceph Storage Cluster requires at least one active
+:doc:`Metadata Server (MDS) daemon </cephfs/add-remove-mds>`, :doc:`creating
+the file system </cephfs/createfs>`, selecting a mount mechanism for clients
+(:doc:`FUSE </cephfs/fuse>` or :doc:`kernel driver </cephfs/kernel>`), and
+configuring :doc:`authentication credentials </cephfs/client-auth>` for
+clients.
 
+For setting up CephFS quickly, checkout the :doc:`CephFS Quick Start document
+</start/quick-cephfs>`.
+
+.. note::
+
+         If :doc:`cephx </rados/configuration/auth-config-ref>` authentication
+         is enabled, a secret key is required for mounting CephFS with FUSE. It
+         is a file containing a key extracted from the keyring.
```

Comment:
Last sentence in note looks redundant since secretfile isn't mentioned anywhere.